arbiter-core/src/environment.rs

```diff
@@ -34,23 +51,30 @@ pub enum State {
     Stopped,
 }
 
+/// The environment struct.
 pub struct Environment {
+    /// label for the environment
     pub label: String,
     pub(crate) state: State,
     pub(crate) evm: EVM<CacheDB<EmptyDB>>,
     pub(crate) tx_sender: TxEnvSender,
-    tx_receiver: TxEnvReceiver,
+    pub tx_receiver: TxEnvReceiver,
```

Comment:
This shouldn't need to be pub, should it? The only one using this should be the EVM that is ran in this module. Things could go bad if an end user accesses this.

Reply:
This was from the merge commit so i just left it

Reply:
Figured you were using it

Reply:
Tests all pass except for the ones with the todo()